src/Header/index.jsx

```diff
@@ -116,6 +117,12 @@ module.exports = React.createClass({
         var cols = props.virtualColumnRendering && props.endColIndex !== null ?
                     props.columns.slice(props.startColIndex, props.endColIndex + 1) :
                     props.columns
+
+        if (props.fixedColumns.length) {
+            _.remove(cols, function(column) { return column.fixed; });
+            cols = props.fixedColumns.concat(cols);
+        }
```

Comment:
This was added to support non-consecutive column freezing by moving them to the beginning. I have removed this now.